packages/lesswrong/components/community/modules/CommunityBanner.tsx

```diff
@@ -5,14 +5,16 @@ import { createStyles } from '@material-ui/core/styles';
 const styles = createStyles((theme: ThemeType): JssStyles => ({
   bannerImg: {
     width: '100vw',
-    height: 200,
+    height: 220,
     backgroundImage: `linear-gradient(to right, ${theme.palette.primary.dark} 20%, transparent), url(https://res.cloudinary.com/cea/image/upload/c_fill,h_380,w_1600,q_auto,f_auto/236225045_2995791057331456_5749161116892625450_n.jpg.jpg)`,
     backgroundSize: 'cover',
     backgroundPosition: 'top right',
-    padding: 40,
+    padding: 50,
```

Comment:
Slightly adjusted the padding to make the page look more balanced without the extra space at the top. The only downside of this is that the page is now a bit short on large screens, but I think it's better to have more of the page devoted to functionality than to pad it out